File: Java/Maths/Calculate the sum of natural numbers using while loop.java
import java.util.Scanner;
class sumOfn_whileLoop {
    public static void main(String args[]) {
        Scanner input = new Scanner(System.in);

        System.out.print("Enter a positive number: ");
        int n = input.nextInt();

        int sum = 0;
        while (n > 0) {
            sum += n;
            n--;
        }
        System.out.println("The sum of nth numbers is = " + sum);
    }
}

File: C++/Maths/Calculate the sum of natural numbers using while loop.cpp
#include <iostream>
using namespace std;

int main(){
    int n, sum = 0;
    cout << "Enter a positive number: ";
    cin >> n;

    while (n > 0) {
        sum += n;
        n--;
    }
    cout << "The sum of nth numbers is = " << sum;
}